packages/plugins/Web3Profile/src/SNSAdaptor/components/Main.tsx

```diff
@@ -16,9 +16,17 @@ export interface MainProps {
 export function Main(props: MainProps) {
     const t = useI18N()
     const { openImageSetting, currentVisitingProfile, accountList } = props
+    if (!accountList)
+        return (
+            <div>
+                <Box marginTop="calc(45% - 47px)">
```

Comment:
The `marginTop` look like magic. Maybe u need to add comment tot explain this.

Reply:
Out of context, reverted.